### PR TITLE
Release 1.16.0

### LINKED ISSUE #154: Release 1.16.0

### PR DESCRIPTION
**Pull request by ocelotl**
Fixes #154

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 ## Unreleased
 
+## 1.16.0
+
+- Mark as deprecated
```
Comment (codeboten) [inline CHANGELOG.md:5]: Is the change to the README what is effectively marking this as deprecated?

Comment (ocelotl) [reply CHANGELOG.md:5]: It's actually [this](https://github.com/lightstep/otel-launcher-python/commit/874258b186209ffeeaf1ee90480acb986154fc18) change.

**Review by codeboten — APPROVED** [review]

**Comment by tuanng-cognite** [comment]
This has not been released yet, hasn't it? I can't find it from pypi.

**Comment by gtmanfred** [comment]
It looks like there was no tag created for this release or for v1.15.0. so the github action to publish those to pypi were never triggered. Could yall please make those tags?  Thanks.

**Comment by ocelotl** [comment]
Just released both :v:

**Comment by gtmanfred** [comment]
Awesome, thank you very much!

**Pull request merged**